buildlib/travis-build

```diff
@@ -20,6 +20,16 @@ cd ../build32
 CC=gcc-6 CFLAGS="-Werror -m32" cmake -GNinja .. -DENABLE_RESOLVE_NEIGH=0
 ninja
 
+# aarch64 build to check compilation on ARM 64bit platform
+cd ../build-aarch64
+if ! (which aarch64-linux-gnu-gcc &> /dev/null); then
+   AARCH64_PATH=`find / -type d -name 'gcc-linaro-*-x86_64_aarch64-linux-gnu' -print -quit 2>/dev/null | xargs echo`
+   export PATH=$PATH:${AARCH64_PATH}/bin
```

Comment:
@jgunthorpe it is not that horrible actually. 

The find is only invoked in the docker environment and even than since the gcc-linaro is unpacked in /, it find it immediately. On the other hand if one day you shuffle the build environment the script will continue to work.